logic/system.js

```diff
@@ -44,6 +44,29 @@ async function getBitcoinP2PHiddenServiceUrl() {
     }
 };
 
+async function getBitcoinRPCConnectionDetails() {
+    try {
+        const [user, hiddenService] = await Promise.all([
+          diskLogic.readUserFile(),
+          diskLogic.readBitcoinRPCHiddenService(),
+        ]);
+        const label = `${user.name}'s Umbrel`;
```

Comment:
Looks like we probably need to encode the label as per Quick Connect's [examples](https://github.com/BlockchainCommons/Gordian/blob/master/Docs/Quick-Connect-API.md#current-format)

```suggestion
        const label = encodeURIComponent(`${user.name}'s Umbrel`);
```

Reply:
Woops, good catch.